src/WSRequestHandler_Sources.cpp

```diff
@@ -1679,6 +1679,40 @@ RpcResponse WSRequestHandler::SetAudioMonitorType(const RpcRequest& request)
 	return request.success();
 }
 
+/**
+* Get the default settings for a given source type.
+*
+* @param {String} `sourceKind` Source name.
+*
+* @api requests
+* @name GetSourceDefaultSettings
+* @category sources
+* @since 4.9.0
+*/
+RpcResponse WSRequestHandler::GetSourceDefaultSettings(const RpcRequest& request)
+{
+	if (!request.hasField("sourceKind")) {
+		return request.failed("missing request parameters");
+	}
+
+    QString sourceKind = obs_data_get_string(request.parameters(), "sourceKind");
```

Comment:
I noticed there a several instances of mixed indents. I noticed our `.editorconfig` may have missed an `indent_size` alongside `indent_style = tab` to work properly. Can you sync with the main branch and see if your editor picks it up?